data/korath/korath missions.txt

```diff
@@ -257,3 +257,81 @@ mission "Small Scale Delivery"
 	on complete
 		payment 1000
 		dialog `You find a Korath who seems to have been waiting for you in the spaceport. They greet you by showing their palms, then outstretch their hands as if waiting to receive the package. You hand off the small delivery and they hand you a small payment of <payment>.`
+
+mission "Efret: Quarg Rescue"
+	boarding
+	repeat
+	invisible
+	deadline 1
+	source
+		government "Kor Efret"
+	on offer
+		"old quarg rep" = "reputation: Quarg"
+		# bring it low enough that capping this will result in war
+		"reputation: Quarg" <?= 1
+		conversation
+			branch "unfriendly quarg" # if you're already at war with them
+				"reputation: Quarg" < 0
+			`The Quarg have brought reinforcements, probably in reaction to a distress call by the <ship>. They hail you, with barely contained anger in their voice. "Remove yourself from hence, and we may yet forgive your aggression. The Kor Efret are under our protection! Think not even about capturing yonder vessel."`
+			choice
+				`  "I had not realized they were under your protection. I will leave at once."`
+				`  (Ignore the warning.)`
+					goto ignore
+				`  "I think you should leave. This is my prize and I'll take what I please."`
+					goto refuse
+			`  "Leave then anon, ere we change our mind."`
+				accept
+			label ignore
+			action
+				"reputation: Quarg" <?= -1
+			`  "So be it. You shall pay for your transgression against the Efret!" The Quarg begin powering up their weapons...`
+				accept
+			label refuse
+			action
+				"reputation: Quarg" <?= -1
+			`  "What impetuous evil is this! Nay, we shall not allow you your plunder; such may not be tolerated." The Quarg open fire before you have a chance to disengage.`
```

Comment:
Did the Quarg destroy the ship? Interesting choice. So, is the idea that they'll just keep destroying any Efreti ship I try to board just to deny me the ship and its outfits? How far are they willing to go with this? Are they going to kill all the Efreti? Or, just destroy all their ships?
Also, why don't they do this in the `ignore` route?

Reply:
Because here you're provocking them but I didnt write this one initially I dont think there should be a 100% destruction it should be a 10% chance at most

Reply:
I'm provoking them into killing all these Efreti? Aren't the Quarg supposed to be disciplined or something?

Reply:
True, maybe they just come with one more ship or stare at you angrily

Reply:
Assuming the player is not bluffing, it's either the Quarg blow up the ship, potentially allowing some Efreti to escape in lifepods, or you explicitly kill them one by one at this point, and they don't want you to be better equipped to kill more in the future.

Reply:
I think the best thing to do is to say "you noticed all Efret mysteriously disapeared before the ship exploded"?